apis/r/R/RcppExports.R

```diff
@@ -105,17 +105,17 @@ check_arrow_array_tag <- function(xp) {
 #' summary(rl)
 #' }
 #' @export
-sr_setup <- function(ctx, uri, colnames = NULL, qc = NULL, dim_points = NULL, dim_ranges = NULL, config = NULL, loglevel = "auto") {
-    .Call(`_tiledbsoma_sr_setup`, ctx, uri, colnames, qc, dim_points, dim_ranges, config, loglevel)
+sr_setup <- function(uri, config, colnames = NULL, qc = NULL, dim_points = NULL, dim_ranges = NULL, loglevel = "auto") {
+    .Call(`_tiledbsoma_sr_setup`, uri, config, colnames, qc, dim_points, dim_ranges, loglevel)
 }
 
-#' @rdname sr_setup
+#' @rdname sr_complete
 #' @export
 sr_complete <- function(sr) {
     .Call(`_tiledbsoma_sr_complete`, sr)
 }
 
-#' @rdname sr_setup
+#' @rdname sr_next
```

Comment:
Ditto. This now creates a new manual page (once you run `roxygenize()`) as you created a new tag.  It was sr_setup on purpose just like the previous one.

Reply:
ditto